test/raw_data_gis/scripts/test_raster_bio2.json

```diff
@@ -13,7 +13,6 @@
   "extract_all": "True",
   "archived": "zip",
   "driver": "GTiff",
-  "licenses": [],
```

Comment:
If no license is to provided, it must simply not be mentioned, providing an empty list results in error when any operation like `citation` is done.